client/landing/stepper/declarative-flow/internals/steps-repository/playground/test/index.tsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import PlaygroundStep from '..';
+import { StepProps } from '../../../types';
+import { renderStep, mockStepProps, RenderStepOptions } from '../../test/helpers';
+
+// Be sure to mock this hook to return true so that the step is rendered.
+jest.mock( 'calypso/landing/stepper/hooks/use-is-playground-eligible', () => ( {
+	useIsPlaygroundEligible: () => true,
+} ) );
+
+const renderPlaygroundStep = (
+	props?: Partial< StepProps >,
+	renderOptions?: RenderStepOptions
+) => {
+	const combinedProps = { ...mockStepProps( props ) };
+
+	return renderStep( <PlaygroundStep { ...combinedProps } />, renderOptions );
+};
+
+const launchButton = () => screen.getByRole( 'button', { name: 'Launch on WordPress.com' } );
+
+describe( 'Playground', () => {
+	describe( 'step', () => {
+		it( 'should render the iframe and the launch button', () => {
+			const { container } = renderPlaygroundStep();
+
+			expect( launchButton() ).toBeInTheDocument();
+			expect( container.querySelector( 'iframe' ) ).toBeInTheDocument();
```

Comment:
When writing RTL tests, we usually prefer:
- `.toBeVisible()` for elements that are supposed to be visible
- `.toBeInTheDocument()` for elements that may be, or are supposed to be invisible, but in the document

Since those appear like something we want visible, should we use `toBeVisible()` instead?

Reply:
Make sense, thanks